src/promptflow-core/promptflow/core/_connection.py

```diff
@@ -170,7 +170,7 @@ def __init__(
         api_base: str,
         api_key: str = None,
         api_type: str = "azure",
-        api_version: str = "2023-07-01-preview",
+        api_version: str = "2024-02-01",
```

Comment:
maybe add some constants for these default api_version, so it's easier to change next time